src/main/java/com/google/devtools/build/lib/bazel/BazelStartupOptionsModule.java

```diff
@@ -33,7 +33,11 @@
         valueHelp = "<path>",
         help =
             "The location of the user .bazelrc file containing default values of "
-                + "Bazel options. If unspecified, Bazel uses the first .bazelrc file it finds in "
+                + "Bazel options. This option can also be specified multiple times.\n"
+                + "E.g. `--bazelrc=x.rc --bazelrc=y.rc` so options in both RCs will be read.\n"
+                + "Note: `--bazelrc x.rc y.rc` is illegal, and each bazelrc file needs to be "
```

Comment:
Would you agree that there is still value for this change because this is the only place that documents `--bazelrc` can used multiple times?

Reply:
Happy to revise if the suggested way is still preferred. I was mostly thinking from other users' point of view, since this is a rather unique use of the options within bazel.

Reply:
I don't think any other flag support `--flag val1 val2` == `--flag=val1 --flag=val2`, which is why I don't think we should suggest such a thing could even be considered possible.

Reply:
Oh I see. Might be due to github's UI. IIUC I will remove 
```
                + "Note: `--bazelrc x.rc y.rc` is illegal, and each bazelrc file needs to be "
```
and leave the rest.